Documentation CYF 1.0/pages/api-events.html

```diff
@@ -138,6 +138,13 @@ <h4>Encounter script events</h4>
                 spareable or not. This event fires <i>after</i> the sparing of monsters is completed. If you spare the
                 last enemy in the encounter, this function will not happen - the encounter is over at that point.
             </p>
+            <p>
+                <br><span class="function new066">HandleFlee(<span class="boolean"> success</span>)</span>
+                Happens when you select the Flee option from the Mercy menu.
```

Comment:
If we follow the last comment, maybe you should add the fact that using this function will prevent the engine's fleeing sequence to run normally!